### PR TITLE
Use faraday

### PR DESCRIPTION
**Pull request by okkez**
This is a first step to use Faraday.
There are no test, yet.

### CODE REVIEW

**Comment by tricknotes** [comment]
Seems good :rocket: 
Is it rebase required?

**Comment by okkez** [comment]
Rebased, just now!

**Comment by tricknotes** [comment]
I'm sure it will be tested in future :ship:

File: lib/daimon_skycrawlers/crawler.rb [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
+    def initialize(url, options = {})
+      @connection = Faraday.new(url, options) do |faraday|
+        if block_given?
+          yield faraday
+        end
+      end
+    end
+
+    # TODO Support POST when we need
+    def fetch(url, params = {})
```
Comment (tricknotes) [inline lib/daimon_skycrawlers/crawler.rb]: Maybe `url` means `path`?

Comment (okkez) [inline lib/daimon_skycrawlers/crawler.rb]: Yes for faraday API.
Not sure for Skycrawlers API.

**Pull request merged**